web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/add-ons.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 'gatsby-wordpress-add-ons'
+title: 'Add Ons'
+slug: '/frontend-starters/nextjs/gatsby/gatsby-wordpress/add-ons'
+sidebar_position: 5
+---
+
+### Advanced Custom Fields
+
+Advanced Custom Fields turns WordPress sites into a fully-fledged content
```

Comment:
```suggestion
The Advanced Custom Fields WordPress plugin, or ACF, turns WordPress sites into a fully-fledged content
```
introduce the full name the first time with the abbreviation, so from here on you can just use 'ACF' instead of spelling out Advanced Custom Fields each time.